tutorials/plot_sensors_time_frequency.py

```diff
@@ -114,15 +114,23 @@
 mne.viz.tight_layout()
 plt.show()
 
+###############################################################################
+# You can also create a joint plot that contains both the aggregated TFR
+# across channels and topomaps at specific times and frequencies.
+
+power.plot_joint(baseline=(-0.5, 0), mode='mean', tmin=-.5, tmax=2,
+                 timefreqs=[(.5, 10), (1.3, 8)])
```

Comment:
why do you need the show?

also on my mac with matplotlib 2.1 I get this:

<img width="670" alt="screen shot 2018-03-17 at 12 38 15" src="https://user-images.githubusercontent.com/161052/37554941-2cde4ea8-29e0-11e8-84bf-18790e2034ad.png">

so tight layout did not work and the I don't see the lines. Can someone replicate?

also when I select a box interactively in the TF plane this pops up a topo that does not look the same as the plot joint above. Any hint?

Reply:
Hm, I saw the missing lines on previous CircleCI runs, but the problem disappeared when I changed a few things.

Don't know about the interactivity. I will have to check what it actually does.

Reply:
The interactive functionality looks well on my end?

<img width="957" alt="screenshot 2018-03-17 12 54 56" src="https://user-images.githubusercontent.com/4321826/37555125-db77f9da-29e2-11e8-88dc-f32d40054250.png">

Or does this look off to you?
(With the exception of choosing Reds over RdBu_r)